From 283cbe7927607fed2471790cf56ee22a76f9db1f Mon Sep 17 00:00:00 2001
From: Chien Nguyen <chien.nguyen.eb@rvc.renesas.com>
Date: Fri, 10 Jun 2016 09:35:04 +0700
Subject: [PATCH] DoorPhone-Modify graph display of memory measurement

At current time, graph display for memory measurement of DoorPhone is not
true due to wrong in formula for memory measurement

This patch file modify formula for memory measurement to make graph display
of memory measurement become true

Signed-off-by: Chien Nguyen <chien.nguyen.eb@rvc.renesas.com>
---
 qml/simpleplayer/GraphDisplay.qml | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/qml/simpleplayer/GraphDisplay.qml b/qml/simpleplayer/GraphDisplay.qml
index 11b1811..aeb449c 100644
--- a/qml/simpleplayer/GraphDisplay.qml
+++ b/qml/simpleplayer/GraphDisplay.qml
@@ -136,11 +136,10 @@ Rectangle{
       ctx.beginPath()
       //top-leftstartpoint
       ctx.moveTo(0,height)
-      //upperline
-
+      //upperline      
       for (i=0;i<21;i++)
       {
-          ctx.lineTo((i*size),(height-root.atemp[i]))
+          ctx.lineTo((i*size),(height-(root.atemp[i] * height /100)))
       }
       ctx.lineTo(width,height)
       //leftlinethroughpathclosing
-- 
1.9.1

